templates/default/slurm/parallelcluster_slurm_cloudbursting.conf.erb

```diff
@@ -0,0 +1,8 @@
+[slurm_cb_config]
+region = us-east-2
+proxy = NONE
+cluster_name = <%= node['cfncluster']['stack_name'].sub("parallelcluster-", "") %>
+max_retry = 3
```

Comment:
do we want to have a global configuration for max_retries?also we might want to set a different value depending on the api we are calling.